media-video/handbrake/handbrake-1.2.2.recipe

```diff
@@ -99,8 +110,10 @@ PATCH()
 
 BUILD()
 {
+	export CPPFLAGS="-I/system/develop/headers/libxml2"
```

Comment:
This is obviously not correct.

Reply:
NOTE: pkgconfig looks correct on x86 and x86_64, yet this bug appeared for this build. Rechecking.

Reply:
diff
-	export CPPFLAGS="-I/system/develop/headers/libxml2" 
+	export CPPFLAGS="`pkgconfig --cflags libxml2`" #or something like that.

Reply:
Fixed.